addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/internal/rest/RestClient.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.mihome.internal.rest;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import javax.net.ssl.SSLContext;
+
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.eclipse.jetty.http.HttpMethod;
+
+/**
+ * A transport layer to execute SSL requests to a given <code>url</code>
+ *
+ * @author Svilen Valkanov
+ */
+public interface RestClient {
+
+    public static final String CERTIFICATE_DIR = "SSL";
+    public static final String CERTIFICATE_FILE_NAME = "miHome.crt";
```

Comment:
>What is this certificate can/will it expire?

Yes, it will expire in six months.

>what then?

... I guess the .crt file should be replaced..

I now it isn't  easy for maintenance, I could try to download the certificate at bundle start..